app/src/androidTest/java/org/mozilla/fenix/ui/CollectionTest.kt

```diff
@@ -42,31 +54,164 @@ class CollectionTest {
+    @Ignore("Temp disable test - see: https://github.com/mozilla-mobile/fenix/issues/5793")
+    // open a webpage, and add currently opened tab to existing collection
+    fun addTabToCollectionTest() {
+        val firstWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+        val secondWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 2)
+
+        createCollection(firstWebPage.url, "testcollection_1")
+
+        // Close the open tab
+        homeScreen {
+            verifyHomeScreen()
+        }
+        Espresso.onView(ViewMatchers.withId(R.id.close_tab_button)).click()
+
+        // On homeview, open another webpage
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(secondWebPage.url) {
+            verifyPageContent(secondWebPage.content)
+        }
+
+        // Save the current page to the testcollection_1
+        navigationToolbar {
+        }.openThreeDotMenu {
+            // click save to collection menu item, type collection name
+            clickBrowserViewSaveCollectionButton()
+            org.mozilla.fenix.ui.robots.mDevice.wait(
+                Until.findObject(By.text("testcollection_1")),
+                TestAssetHelper.waitingTime)
+            Espresso.onView(ViewMatchers.withText("testcollection_1")).click()
+            Thread.sleep(5000)
+            mDevice.pressBack() // go to main page
+        }
+
+        // close currently opened tab
+        homeScreen {
+            verifyHomeScreen()
+            Espresso.onView(ViewMatchers.withId(R.id.close_tab_button)).click()
+            org.mozilla.fenix.ui.robots.mDevice.wait(
+                Until.findObject(By.text("testcollection_1")),
+                TestAssetHelper.waitingTime)
+            // On homeview, expand the collection and open the first saved page
+            Espresso.onView(ViewMatchers.withText("testcollection_1")).click()
+            Espresso.onView(ViewMatchers.withText("Test_Page_1")).click()
+        }
+        // Page content: 1
+        browserScreen {
+            verifyPageContent("Page content: 1")
+            mDevice.pressBack() // go to main page
+        }
+
+        // tab_in_collection_item
+        homeScreen {
+            verifyHomeScreen()
+            Espresso.onView(ViewMatchers.withId(R.id.close_tab_button)).click()
+            // On homeview, expand the collection and open the first saved page
+            org.mozilla.fenix.ui.robots.mDevice.wait(
+                Until.findObject(By.text("Test_Page_2")),
+                TestAssetHelper.waitingTime)
+            Espresso.onView(ViewMatchers.withText("Test_Page_2")).click()
+        }
+
+        // Page content: 2
+        browserScreen {
+            verifyPageContent("Page content: 2")
+        }
     }
 
-    @Ignore
     @Test
-    fun OpenTabFromCollectionTest() {
-        // Open one tab from Collection in the Homescreen view
+    @Ignore("Temp disable test - see: https://github.com/mozilla-mobile/fenix/issues/5793")
```

Comment:
I guess all these disabled tests work on physical device, just not on x86 (for now) at least til #5793 is resolved, true?

Reply:
Yes they work on physical devices when ran locally.